src/main/WikiVectors.java

```diff
@@ -46,7 +46,7 @@
  * as a precursor for indexing vectors in benchmark runs. It's provided for "offline" (manual) use,
  * and doesn't factor into benchmark execution.
  */
-public class WikiVectors<T> {
+public class WikiVectors {
```

Comment:
I couldn't find `T` used anywhere. I think it was added a ways back thinking we would use generics for `VectorDictionary` as well. So, I removed it for now.